molecule/amq_upgrade/prepare.yml

```diff
@@ -7,4 +7,18 @@
         name:
           - sudo
           - iproute
-        state: present
\ No newline at end of file
+        state: present
+    - name: Check local download archive path
+      ansible.builtin.stat:
+        path: "{{ lookup('env', 'PWD') }}"
+      register: local_path
+      delegate_to: localhost
+    - name: Check archive for 7.9.4
+      ansible.builtin.stat:
+        path: "{{ local_path.stat.path }}/amq-broker-7.9.4-bin.zip"
+      register: local_archive_path
+      delegate_to: localhost
+    - name: Include AMQ role for 7.9
+      ansible.builtin.include_role:
+        name: amq_broker
+      when: local_archive_path.stat.exists
```

Comment:
@guidograzioli yet another opportunity for the exists jinja filter!

Reply:
will open a PR